src/entry.ts

```diff
@@ -24,5 +38,16 @@ export * from "@/composables/index"
 // each can be registered via Vue.component()
 export * from "@/lib-components/index"
 
-export { BaseAPI }
-export type { RequestOptions }
+// Http Client exports
+export { BaseAPI, hasHttpErrStatus, isHttpCancel, isHttpError }
+export type {
```

Comment:
You'll start to see this a lot more.  Seems to be a hallmark of better libraries to export types that are expected to be consumed.  Vue for example lets you `import {Ref, ComputedRef, etc...} from "vue"`.  Trees should do the same, so you don't need to import from a nested `types` directory.